zk_ee/src/common_structs/history_map.rs

```diff
@@ -1,13 +1,10 @@
 //! Contains a key-value map that allows reverting items state.
```

Comment:
nit: do you think it makes sense to split this file into few? Maybe tests and struct for inner elements separately?

At least it will be easier to review